src/icons/smart_goals.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="22px" height="18px" viewBox="0 0 22 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
All icons should have a width, height and viewbox of 24px, this must have been exported from Sketch incorrectly.

```
width="24px" height="24px" viewBox="0 0 24 24"
```